src/core/security/security_context.h

```diff
@@ -36,6 +36,10 @@
 
 #include "src/core/security/credentials.h"
 
+#ifdef __cplusplus
```

Comment:
My hope was that you didn't need to include this file. `grpc_security.h` should be sufficient but let me have a closer look.